modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -0,0 +1,329 @@
+			ObjectDefinitionAPI.staticStudentObjectId();
+
+			ObjectDefinitionAPI.staticSubjectObjectId();
+		}
+
+		task ("And Given many-to-many studentsSubjects relationship created") {
+			ObjectDefinitionAPI.createRelationshipWithObjectDefinitionNames(
+				childObjectName = "Subject",
+				deletionType = "cascade",
+				name = "studentsSubjects",
+				parentObjectName = "Student",
+				type = "manyToMany");
+		}
+
+		task ("And Given one-to-many accountStudents relationship created") {
+			var accountId = ObjectDefinitionAPI.getObjectDefinitionIdByName(name = "AccountEntry");
+
+			ObjectDefinitionAPI.createRelationship(
+				deletionType = "cascade",
+				en_US_label = "AccountStudents",
+				name = "accountStudents",
+				objectDefinitionId1 = ${accountId},
+				objectDefinitionId2 = ${staticStudentObjectId},
+				type = "oneToMany");
+		}
+
+		task ("And Given object entries of Student, Subject, Account created with related entries") {
+			var studentAccountId = AccountAPI.createAccount(name = "accountForStudent");
+
+			CustomObjectAPI.createStudentRelatedToAccount(
+				accountId = ${studentAccountId},
+				name = "Able");
+
+			ObjectDefinitionAPI.createObjectEntryWithName(
+				en_US_plural_label = "subjects",
+				name = "English");
+
+			ObjectDefinitionAPI.relateObjectEntriesByName(
+				child_plural_label = "subjects",
+				childEntryName = "English",
+				parent_plural_label = "students",
+				parentEntryName = "Able",
+				relationshipName = "studentsSubjects");
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		BatchPlanner.batchPlannerTearDown();
+
+		AccountAPI.tearDownAllAccounts();
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 4
+	test ActionsNodeAreAvailableInDonwloadedFile {
+		task ("And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("When exporting file in JSONL format with all fields selected") {
+			ImportExport.configureExport(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				exportFields = "Attribute Code",
+				exportFileFormat = "JSONL");
+
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+		}
+
+		task ("Then 'actions' node is available in the dowloaded file") {
+			var response = ObjectDefinitionAPI.getObjectEntries(en_US_plural_label = "students");
+
+			var entryId = JSONPathUtil.getProperty(
+				property = "$..items..[0]id",
+				response = ${response});
+
+			ImportExport.unzipDownloadedExportFile();
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
```

Comment:
Instead of assert only the actions.delete.href, I think it's better to assert all actions in actions node